scout/server/blueprints/cases/controllers.py

```diff
@@ -240,8 +240,10 @@ def _populate_acmg(evaluated_variants):
     Returns:
 
     """
+
     for variant in evaluated_variants:
-        if isinstance(variant.get("acmg_classification"), int):
+
+        if not isinstance(variant, str) and isinstance(variant.get("acmg_classification"), int):
```

Comment:
Nice! Let's put in on stage, we could try with the same variant that crashed!

Reply:
Its there already! 😁